network.sh

```diff
@@ -188,6 +188,11 @@ if  [[ ! -z "${INSTALLER_PROXY:-}" ]]; then
   fi
 fi
 
+if [ -n "$NETWORK_CONFIG_FOLDER" ]; then
```

Comment:
`network.sh: line 191: NETWORK_CONFIG_FOLDER: unbound variable
make: *** [Makefile:42: ironic] Error 1
`
04_setup_ironic.sh has "set -u" and it sources network.sh
replace with `"${NETWORK_CONFIG_FOLDER:-}"`

Reply:
Oops, thanks. Fixed.